src/hvm.rs

```diff
@@ -1317,7 +1325,13 @@ impl Runtime {
   }
 
   pub fn alloc_term_from_code(&mut self, code: &str) -> u128 {
-    self.alloc_term(&read_term(code).1)
+    let term = read_term(code);
+    match term {
+      Ok((.., term)) => {
+        return self.alloc_term(&term);
+      }
+      Err(err) => 0 // TODO: what should we do here?
```

Comment:
What should we do here? This function is not used, maybe we should just remove it? @quleuber